src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "pch.h"
 #include "Terminal.hpp"
+#include <climits>

```

Comment:
Can this guy go into `LibraryIncludes.h` with the other similar <c*> headers?

Reply:
Done. I didn't do it originally because I didn't want to change that file, which gets included almost everywhere including in a lot of precompiled headers.